### PR TITLE
Changes for October 2021

### PR DESCRIPTION
**Pull request by lanceleonard**
- Initial draft for October 2021 changes.

### CODE REVIEW

**Review by kaitlynmichael — APPROVED** [review]

File: changelog.md [hunk 1]
```diff
@@ -2,6 +2,45 @@
 
 The docs are updated regularly; here are some recent highlights:
 
+## October 2021
+
+- Redis Enterprise Software doc updates: 
+
+    - Updated docs to support the v6.2.8 release ([PR #1632](https://github.com/RedisLabs/redislabs-docs/pull/1632)), which included:
+        - [Release notes](https://docs.redis.com/latest/rs/release-notes/rs-6-2-8-october-2021/) ([PR #1628](https://github.com/RedisLabs/redislabs-docs/pull/1632))
```
Comment (rrelledge) [inline changelog.md:10]:
```suggestion
        - [Release notes](https://docs.redis.com/latest/rs/release-notes/rs-6-2-8-october-2021/) ([PR #1628](https://github.com/RedisLabs/redislabs-docs/pull/1628)) 
```

File: changelog.md [hunk 2]
```diff
@@ -2,6 +2,45 @@
...
+        - [Release notes](https://docs.redis.com/latest/rs/release-notes/rs-6-2-8-october-2021/) ([PR #1628](https://github.com/RedisLabs/redislabs-docs/pull/1632)) 
+        - Updates to the [Redis upgrade policy](https://docs.redis.com/latest/rs/installing-upgrading/upgrading/#redis-upgrade-policy) docs ([PR #1599](https://github.com/RedisLabs/redislabs-docs/pull/1599))
+        - Updates to the [backup data](https://docs.redis.com/latest/rs/administering/import-export/database-backup/) and [export data](https://docs.redis.com/latest/rs/administering/import-export/exporting-data/) articles ([PR #1548](https://github.com/RedisLabs/redislabs-docs/pull/1548) & [PR #1598](https://github.com/RedisLabs/redislabs-docs/pull/1598))
+
+    - REST API docs are [in preview](https://docs.redis.com/staging/rs-rest-api/rs/references/rest-api/) ([PR #1533](https://github.com/RedisLabs/redislabs-docs/pull/1533), [PR #1621](https://github.com/RedisLabs/redislabs-docs/pull/1621))
+
+- Redis Enterprise Cloud updates:
+
+    - REST API docs have been clarified and updated to reflect the new user experience ([PR #1651](https://github.com/RedisLabs/redislabs-docs/pull/1615)), which includes:
```
Comment (rrelledge) [inline changelog.md:18]:
```suggestion
    - REST API docs have been clarified and updated to reflect the new user experience ([PR #1615](https://github.com/RedisLabs/redislabs-docs/pull/1615)), which includes:
```

File: changelog.md [hunk 3]
```diff
@@ -2,6 +2,45 @@
...
+
+- Additional changes
+
+    - Modules: A new article documents Redis Enterprise [feature compatibility support](https://docs.redis.com/latest/modules/enterprise-capabilities/) ([PR #1504](https://github.com/RedisLabs/redislabs-docs/pull/1504))
+
+    - Release notes have been added for:
+        - [RedisGraph v2.4.11](https://docs.redis.com/latest/modules/redisgraph/release-notes/redisgraph-2.4-release-notes/#v2411-october-2021) ([PR #1560](https://github.com/RedisLabs/redislabs-docs/pull/1560))
+        - [RediSearch v2.0.12](https://docs.redis.com/latest/modules/redisearch/release-notes/redisearch-2.0-release-notes/#v2012-september-2021) ([PR #1616](https://github.com/RedisLabs/redislabs-docs/pull/1616))
+
+    - RedisInsight: [v1.11 release notes](https://docs.redis.com/latest/ri/release-notes/v1.11.0/) for have been added ([PR #1605](https://github.com/RedisLabs/redislabs-docs/pull/1605))
```
Comment (rrelledge) [inline changelog.md:37]:
```suggestion
    - RedisInsight: [v1.11 release notes](https://docs.redis.com/latest/ri/release-notes/v1.11.0/) have been added ([PR #1605](https://github.com/RedisLabs/redislabs-docs/pull/1605))
```

**Review by rrelledge — APPROVED** [review]

File: changelog.md (hunk 3 above)
Comment (lanceleonard) [reply changelog.md:37]: Fixed.

File: changelog.md (hunk 2 above)
Comment (lanceleonard) [reply changelog.md:18]: Fixed.

File: changelog.md (hunk 1 above)
Comment (lanceleonard) [reply changelog.md:10]: Fixed.

**Pull request merged**